ssl/statem/statem_local.h

```diff
@@ -31,6 +31,9 @@
 /* Max ServerHello size permitted by RFC 8446 */
 #define SERVER_HELLO_MAX_LENGTH         65607
 
+/* Max CertificateVerify size permitted by RFC 8446 */
+#define CERTIFICATE_VERIFY_MAX_LENGTH   65538
```

Comment:
I calculate this as 65539...am I wrong?

````
      struct {
          SignatureScheme algorithm;
          opaque signature<0..2^16-1>;
      } CertificateVerify;
````

`2^16 - 1` == 65535
plus 2 "length" bytes for the signature
plus 2 byes to identify the signature scheme
== 65539

Reply:
Good catch! I will update it.